src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/LanguageServerTestBase.cs

```diff
@@ -44,6 +49,48 @@ public LanguageServerTestBase()
 
         protected ILoggerFactory LoggerFactory { get; }
 
+        protected static async Task<(CSharpTestLspServer Server, Uri DocumentUri)> CreateCSharpTestLspServerAsync(
```

Comment:
It seems weird to return a `documentUri` when you create the server. It feels like that's kinda encoding that there could only be one document tracked by this server. What if `CreateCSharpTestLspServerAsync` took an `IEnumerable<(string DocumentUri, SourceText CSharpSourceText)>`? That would let the caller have more control of the document URI (if they care) and make the return type here a bit simpler.

Reply:
Ah, good point! Updated.